src/main/kotlin/org/quiltmc/community/modes/quilt/extensions/logs/parsers/QSLVersionParser.kt

```diff
@@ -18,7 +18,8 @@ import kotlinx.serialization.json.Json
 import org.quiltmc.community.modes.quilt.extensions.logs.misc.ModrinthVersion
 import kotlin.time.Duration.Companion.minutes
 
-private val MATCH_REGEX = "- quilted_fabric_api ([^\\n]+)".toRegex(RegexOption.IGNORE_CASE)
+private val QSL_MATCH_REGEX = "- quilted_fabric_api ([^\\n]+)".toRegex(RegexOption.IGNORE_CASE)
+private val MINECRAFT_MATCH_REGEX = "- minecraft ([^\\n]+)".toRegex(RegexOption.IGNORE_CASE)
 private const val MODRINTH_URL = "https://api.modrinth.com/v2/project/qsl/version"
```

Comment:
You can add the filter to the URL itself, y'know. <https://api.modrinth.com/v2/project/qsl/version?game_versions=["1.19"]>

Reply:
Changed in ea1f0a2. I have to say, this isn't the easiest format to work with, but it works.